### PR TITLE
python3Packages.pyhocon: 0.3.53 -> 0.3.58

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Update to latest upstream release 0.3.58

Change log:
- https://github.com/chimpler/pyhocon/releases/tag/0.3.58

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 130158` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python38Packages.pyhocon</li>
    <li>python39Packages.pyhocon</li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 130158` at a107c613 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.pyhocon</li>
    <li>python39Packages.pyhocon</li>
  </ul>
</details>

File: pkgs/development/python-modules/pyhocon/default.nix [hunk 1]
```diff
@@ -1,38 +1,38 @@
...
-  # See https://github.com/chimpler/pyhocon/pull/203
-  doCheck = false;
+  pythonImportsCheck = [ "pyhocon" ];
 
   meta = with lib; {
     homepage = "https://github.com/chimpler/pyhocon/";
     description = "HOCON parser for Python";
-    # Long description copied from
-    # https://github.com/chimpler/pyhocon/blob/55a9ea3ebeeac5764bdebebfbeacbf099f64db26/setup.py
-    # (the tip of master as of 2019-03-24).
```
Comment (chreekat) [inline pkgs/development/python-modules/pyhocon/default.nix:35]: Please keep this attribution - it's the least we can do. :)

**Review by chreekat — APPROVED** [review]
Thanks! Particularly thank you for enabling tests.

I have not done many reviews with Github's newer review features. I'm selecting the 'Approve' option although I'd prefer to see the attribution comment restored.

File: pkgs/development/python-modules/pyhocon/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/development/python-modules/pyhocon/default.nix:35]: I don't get why we should keep the comment. It seems to be the same source as for the other details, e.g. the same URL where the source is coming. 

I haven't seen any other derivation that is doing this.

Comment (chreekat) [reply pkgs/development/python-modules/pyhocon/default.nix:35]: Well, I can tell you why I put it there in the first place. The long description is a verbatim quote, and as a diligent author, I cited it. Doing so reflects well on me and it's useful information anyway. If other packages are using verbatim quotes, perhaps they should cite them as well. 

I'd like to turn the question around, though. How does removing this citation help you achieve a version bump?

Comment (SuperSandro2000) [reply pkgs/development/python-modules/pyhocon/default.nix:35]: That is literally the description on https://pypi.org/project/pyhocon/

We normally don't quote those things. The description might be adapted over time or changed to follow NixOS standards. Having a 3 line comment about this is unnecessary and just noise.

File: pkgs/development/python-modules/pyhocon/default.nix [hunk 2]
```diff
@@ -1,38 +1,38 @@
...
     longDescription = ''
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pyhocon/default.nix:36]:
```suggestion
    # taken from https://pypi.org/project/pyhocon/
    longDescription = ''
```

**Pull request merged**